app/src/pages/b/[businessId].tsx

```diff
@@ -39,7 +39,7 @@ const getPageContentByBusinessSlug = async (slug: string) => {
       longitude: page?.custom_fields?.longitude,
       instagram: page?.custom_fields?.instagram,
       website: page?.custom_fields?.website,
-      marker_icon: page?.custom_fields?.marker_icon,
+      markerIcon: page?.custom_fields?.markerIcon,
```

Comment:
Let's keep the wordpress fields as: `marker_icon`, but the fields in the typescript objects as camelcase: `markerIcon`